data/json/recipes/recipe_deconstruction.json

```diff
@@ -3478,15 +3485,15 @@
   {
     "type": "uncraft",
     "result": "whisk",
-    "time": "1 m 30 s",
+    "time": "30 s",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "scrap", 3 ] ] ]
   },
   {
     "result": "wine_glass",
     "type": "uncraft",
-    "time": "30 s",
-    "components": [ [ [ "glass_shard", 4 ] ] ]
+    "time": "1 s",
+    "components": [ [ [ "glass_shard", 2 ] ] ]
```

Comment:
wine glass volume is 48ml lower than it should be based upon the glass here.

Reply:
It seems that there is a lot of inconsistencies if we are too strict with the volume/weight ratio in almost every item. For example the sheet_metal_small has the exact same volume from id: "scrap" though both are probably from "pure steel" and the sheet_metal_small has the weight of 5 metal scraps.

Basically I think that there should be a "minimum volume" considering the most dense structure from a material, but it's volume can be bigger depending if we take in consideration the item's structure (if it has hollow parts inside of it for example) and how the volume is calculated.

The wine glass now should be 262ml due it being able to hold 250ml + the volume of 2 glass shards that have 30ml each, then it has 2ml of wasted volume due it's shape/etc. It's weight is 161g, 150 due the 2 glass shards and the rest from paint/plastic/etc since we mostly find them when looting houses.

Anyway it wasn't me who created those, I am just making the change so we can break/disassemble them. I am just roleplaying and interpreting what would it be. Of course this can be changed anytime.

Reply:
> It seems that there is a lot of inconsistencies if we are too strict with the volume/weight ratio in almost every item.

Personally I think this is a symptom of a larger problem with consistency in recipes/items. For example, take the swim fins. They cost 8 plastic chunks (2000 ml of solid plastic) and yield 1250ml of fin.

> The wine glass now should be 262ml due it being able to hold 250ml + the volume of 2 glass shards that have 30ml each, then it has 2ml of wasted volume due it's shape/etc.

Exactly! But many items like the pan or the ceramic mug/plate/bowl/dish don't even fit into these basic guidelines.

Reply:
So now are we ready to merge? Cause I looked up the the ceramic mug and after the change I made it seems ok with RL, being able to hold 250ml but should have a volume of 375ml and a mass of 375g. I didn't search what the other things should be.

But it seems a bit excessive that the most basic ceramic component have a 250ml volume and 750g mass (ceramic shard). But I have no idea what changing this can impact and what changes will need to be made. Maybe creating a new component?

But I just included the ceramic_mug disassemble recipe since the others already existed and the mug had roughly the same mass from the others. Then should we disable all of them? But I think that we need some simple items that can be a source for the ceramic and without the recipes those things will become completely useless and indestructible.

Then can we merge it like this till it is made a overhaul in the ceramic tree?

Reply:
Little bit late, but yeah, ceramic needs an overhaul. What's there is fine.